### PR TITLE
s/properPlacement/improperPlacement/

### PR DESCRIPTION
**Pull request by calbach**
Depends on merge of https://github.com/ga4gh/schemas/pull/561.

A few things I couldn't immediately figure out from the docs:

- How to run the affected tests locally? I guess this will be run by travis when I send the PR anyways.
- How to pull in the latest schemas, _protocol_definitions.py appears to be autogenerated (for now, I've just manually edited it).

### CODE REVIEW

**Comment by david4096** [comment]
Hi @calbach !

To generate schemas: `python scripts/process_schemas.py -i ../schemas/src/main/resources/avro 0.6.0a2`. I only see reference to properPlacement in that one test: `nosetests tests.datadriven.test_reads`

File: ga4gh/_protocol_definitions.py [hunk 1]
```diff
@@ -3071,25 +3070,25 @@ def __init__(self, **kwargs):
 postMethods = \
     [('/callsets/search',
       SearchCallSetsRequest,
-      SearchCallSetsResponse),
+      SearchReadsResponse),
```
Comment (calbach) [inline ga4gh/_protocol_definitions.py]: This looks wrong. Bug in the generation script?

**Comment by calbach** [comment]
Thanks, I ran the provided command and things mostly worked - see comment inline for the only issue I noticed.

File: ga4gh/_protocol_definitions.py (hunk 1 above)
Comment (dcolligan) [inline ga4gh/_protocol_definitions.py]: Hm, couldn't reproduce this.  I'll create a PR with your changes minus the erroneous diff in postMethods.

**Comment by dcolligan** [comment]
Just created a new PR here: #960

Closing this one...

**Pull request closed without merging**